runelite-api/src/main/java/net/runelite/api/SoundEffectID.java

```diff
@@ -62,7 +62,33 @@
 	public final static int ZERO_DAMAGE_SPLAT = 511;
 	public final static int TAKE_DAMAGE_SPLAT = 510;
 	public final static int ATTACK_HIT = 2498;
-	public final static int PRAYER_ACTIVATE_VROOM = 2690;
+
+	public final static int PRAYER_ACTIVATE_2690 = 2690; //Thick Skin
```

Comment:
Instead of naming each of these after the ID, name them after the prayer, eg. `PRAYER_ACTIVATE_THICK_SKIN`